src/EPR.Accreditation.Facade/Services/AccreditationService.cs

```diff
@@ -287,5 +287,33 @@ public async Task<HasOverseasAgentDto> GetHasOverseasAgent(Guid id)
 
             await _httpAccreditationService.UpdateAccreditation(id, accreditation);
         }
+
+        /// <summary>
+        /// Gets PRN tonnage data for the given accreditation.
+        /// </summary>
+        /// <param name="accreditationExternalId">Accreditation Id.</param>
+        /// <returns>DTO containing PRN tonnage data.</returns>
+        public async Task<PrnTonnesPlannedDto> GetPrnTonnesPlanned(Guid accreditationExternalId)
+        {
+            var accreditation = await _httpAccreditationService.GetAccreditation(accreditationExternalId);
+            var dto = _mapper.Map<PrnTonnesPlannedDto>(accreditation);
+            dto.PrnPlannedTonnesType = accreditation.Large == null ? null : accreditation.Large.Value ? PrnPlannedTonnesType.Over : PrnPlannedTonnesType.Upto;
+            return dto;
+        }
+
+        /// <summary>
+        /// Updates PRN tonnage data for the given accreditation.
+        /// </summary>
+        /// <param name="accreditationExternalId">Accreditation Id.</param>
+        /// <param name="prnTonnesPlannedDto">DTO containing PRN tonnage data.</param>
+        /// <returns>Completed Task.</returns>
+        public async Task UpdatePrnTonnesPlanned(
+            Guid accreditationExternalId, 
+            PrnTonnesPlannedDto prnTonnesPlannedDto)
+        {
+            var accreditation = new Common.Dtos.Accreditation { LargeFee = prnTonnesPlannedDto.PrnPlannedTonnesFee };
```

Comment:
Even though it's just one property, put the braces and property all on individual lines. It just reads a bit better that way.